server/middleware/setupFeatureFlags.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestHandler, Router } from 'express'
+import { Services } from '../services'
+import FeatureFlagService, { FeatureFlag } from '../services/featureFlagService'
+
+export const featureFlagsToUse: Array<FeatureFlag> = ['allow-sufficient-information-request-without-confirmation']
+
+export const setupFeatureFlags = ({ featureFlagService }: Services): Router => {
+  const router = Router()
+  router.use(populateFeatureFlags(featureFlagService))
+  return router
+}
+
+export const populateFeatureFlags = (featureFlagService: FeatureFlagService): RequestHandler => {
+  return async (_, __, next) => {
+    try {
+      featureFlagsToUse.forEach(async flag => {
+        const flagBool = await featureFlagService.getBooleanFlag(flag)
```

Comment:
@pezholio I think we might want to throttle this. Unsure of a suitable delay, maybe somewhere between 1 and 5 minutes?
https://underscorejs.org/#throttle

Reply:
I have added the throttle on line 10

Reply:
Will this run on every request, or can we ensure it only runs when we want it?

Reply:
Now the throttling is in place I believe it will only run every 1.5 seconds, but will test to confirm